content/docs/atom-guide/dev-guide/shaders/azsl/srg-semantics.md

```diff
@@ -1,36 +1,38 @@
 ---
 linktitle: SRG Semantics
-title: AZSL, Shader Resource Group Semantics (ShaderResourceGroupSemantic).
-description: Learn about AZSL Shader Resource Group Semantics in the Atom Renderer.
+title: Shader Resource Group Semantics (ShaderResourceGroupSemantic).
+description: Learn about Amazon Shading Language (AZSL) shader resource group semantics (SRG) for Atom Renderer.
 weight: 100
 ---
 
 The `ShaderResourceGroupSemantic`, SRG Semantic, for short, defines the order in which descriptor bindings and descriptor sets (descriptor sets are also known as register spaces in HLSL) are emitted for each `ShaderResourceGroup`.  
```

Comment:
```suggestion
The Shader Resource Group Semantic (SRG Semantic), for short, defines the order in which descriptor bindings and descriptor sets are emitted for each SRG. In HLSL, descriptor sets are also known as _register spaces_.   In AZSL, define SRGs by using the `ShaderResourceGroupSemantic` class. 
```
* Consider the following suggestion to separate the "concept" of SRGs from the code syntax of defining an SRG.

Reply:
Fixed.